src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.cs

```diff
@@ -107,7 +109,7 @@ void IObjectWritable.WriteTo(ObjectWriter writer)
             }
         }
 
-        protected override string GetText(IFormatProvider formatProvider)
+        protected override string? GetText(IFormatProvider? formatProvider)
```

Comment:
❔ Did we really intend for this to be allowed to return `null`?

Reply:
Good catch. Looks like I got over-eager here with a different case.

---
In reply to: [356231402](https://github.com/dotnet/roslyn/pull/40268#discussion_r356231402) [](ancestors = 356231402)